plugins/chat/app/controllers/chat/chat_controller.rb

```diff
@@ -412,6 +412,7 @@ def preloaded_chat_message_query
           .includes(:uploads)
           .includes(chat_channel: :chatable)
           .includes(:thread)
+          .includes(:chat_mentions)
```

Comment:
You probably need to add it chat messages query following recent @martin changes

Reply:
It would be nice if in such cases we refactor logic in place instead of creating a full duplicate of the logic. I'll add this to the new query, but I'm going to need to also make sure that everything is works as expected and properly tested.

Reply:
We just can't all remove at once

Reply:
Sure feel free to do so, we will remove this controller endpoint soon, but no problem from me if you want to do this :+1: